openrtx/src/core/graphics.c

```diff
@@ -210,6 +210,18 @@ void gfx_fillScreen(color_t color)
     }
 }
 
+color_t gfx_getPixel(point_t pos)
+{
+    if (pos.x >= CONFIG_SCREEN_WIDTH || pos.y >= CONFIG_SCREEN_HEIGHT ||
```

Comment:
Please isolate each logic condition within parenthesis. Is not _logically_ useful, but it improves readability